Samples/CameraSettingsExternalSettingsApp/DefaultControlHelper/DefaultControlHelper.h

```diff
@@ -15,32 +15,27 @@ namespace winrt::DefaultControlHelper::implementation
     struct DefaultControlManager : DefaultControlManagerT<DefaultControlManager>
     {
         DefaultControlManager(winrt::hstring cameraSymbolicLink);
+        ~DefaultControlManager();
 
         winrt::DefaultControlHelper::DefaultController CreateController(DefaultControllerType type, const uint32_t id);
 
-        wil::com_ptr_t<IMFCameraControlDefaultsCollection> GetCollection() { return m_controlDefaultsCollection; }
+        wil::com_ptr_t<IMFCameraControlDefaultsCollection> GetDefaultCollection();
 
-        HRESULT Save();
+        HRESULT SaveDefaultCollection(IMFCameraControlDefaultsCollection* collection);
 
     private:
 
-        wil::com_ptr_t<IMFCameraConfigurationManager> m_configManager;
-        wil::com_ptr_t<IMFCameraControlDefaultsCollection> m_controlDefaultsCollection;
+        wil::com_ptr_t<IMFCameraConfigurationManager> m_spConfigManager;
+        wil::com_ptr<IMFAttributes> m_spAttributes;
     };
 
     struct DefaultController : DefaultControllerT<DefaultController>
     {
         DefaultController(winrt::DefaultControlHelper::DefaultControllerType type, uint32_t id, winrt::com_ptr<DefaultControlManager> manager);
-        bool HasDefaultValueStored() { return m_hasDefaultValueStored; }
-        int32_t DefaultValue();
-        void DefaultValue(int32_t const& value);
+        winrt::Windows::Foundation::IReference<int32_t> TryGetDefaultValueStored();

```

Comment:
nit: for my ear this sounds somehow backward wording.
TryGetDefaultValue
TryGetStoredDefaultValue